docs/getting_started/getting_started_chat.md

```diff
@@ -0,0 +1,217 @@
+\
+Chat models are a variation on language models. While chat models use language models under the hood, the interface they expose is a bit different: rather than expose a "text in, text out" API, they expose an interface where "chat messages" are the inputs and outputs.\
+\
+Chat model APIs are fairly new, so we are still figuring out the correct abstractions.\
+\
+**NOTE**: Before going further please make sure you've followed the [Installation](./getting_started.md#installation) and [Environment Setup](./getting_started.md#environment-setup) steps from the Quickstart Guide.
+
+## Message Completions
+
+You can get chat completions by passing one or more messages to the chat model. The response will be a message. The types of messages currently supported in LangChain are `AIMessage`, `HumanMessage`, `SystemMessage`, and `ChatMessage` -- `ChatMessage` takes in an arbitrary role parameter. Most of the time, you'll just be dealing with `HumanMessage`, `AIMessage`, and `SystemMessage`.
+
+```python
+from langchain.chat_models import ChatOpenAI
+from langchain.schema import (
+    AIMessage,
+    HumanMessage,
+    SystemMessage
+)
+
+chat = ChatOpenAI(temperature=0)
+```
+
+You can get completions by passing in a single message.
+
+```python
+chat([HumanMessage(content="Translate this sentence from English to French. I love programming.")])
+```
+
+```pycon
+AIMessage(content="J'aime programmer.", additional_kwargs={})
+```
+
+You can also pass in multiple messages for OpenAI's gpt-3.5-turbo and gpt-4 models.
+
+```python
+messages = [
+    SystemMessage(content="You are a helpful assistant that translates English to French."),
+    HumanMessage(content="I love programming.")
+]
+chat(messages)
+# -> AIMessage(content="J'aime programmer.", additional_kwargs={})
+```
+
+You can go one step further and generate completions for multiple sets of messages using `generate`. This returns an `LLMResult` with an additional `message` parameter:
+```python
+batch_messages = [
+    [
+        SystemMessage(content="You are a helpful assistant that translates English to French."),
+        HumanMessage(content="I love programming.")
+    ],
+    [
+        SystemMessage(content="You are a helpful assistant that translates English to French."),
+        HumanMessage(content="I love artificial intelligence.")
+    ],
+]
+result = chat.generate(batch_messages)
+result
+# -> LLMResult(generations=[[ChatGeneration(text="J'aime programmer.", generation_info=None, message=AIMessage(content="J'aime programmer.", additional_kwargs={}))], [ChatGeneration(text="J'aime l'intelligence artificielle.", generation_info=None, message=AIMessage(content="J'aime l'intelligence artificielle.", additional_kwargs={}))]], llm_output={'token_usage': {'prompt_tokens': 57, 'completion_tokens': 20, 'total_tokens': 77}})
```

Comment:
Could do the ```pycon syntax here right?

Reply:
yup, couple places where that's the case, was gonna save for sep pr

Reply:
Maybe rename `getting_started.md` to `getting_started_llm.md`, so we have balanced names together with `getting_started_chat.md`? LOGAF low